aws_lambda_builders/workflows/custom_make/workflow.py

```diff
@@ -23,7 +23,7 @@ class CustomMakeWorkflow(BaseWorkflow):
 
     EXCLUDED_FILES = (".aws-sam", ".git")
 
-    BUILD_IN_SOURCE_BY_DEFAULT = False
+    DEFAULT_BUILD_DIR = BuildDirectory.SCRATCH
```

Comment:
by making this change, we can remove the logic of choosing a working/build directory based on the `build_in_source` parameter (see `_select_working_directory` below). Similarly, in other workflows we won't have to define this kind of logic either as the parent Workflow class now handles setting the `build_dir`.